pycqed/measurement/calibration_toolbox.py

```diff
@@ -583,34 +598,32 @@ def mixer_skewness_cal_UHFQC_adaptive(UHFQC, SH, source, AWG,
     # Loads a train of pulses to the AWG to trigger the UHFQC continuously
     AWG.stop()
     st_seqs.generate_and_upload_marker_sequence(
-                    5e-9, 1.0e-6, RF_mod=False,
-                    acq_marker_channels=acquisition_marker_channel)
+        5e-9, 1.0e-6, RF_mod=False,
+        acq_marker_channels=acquisition_marker_channel)
     AWG.run()
 
     #  Ensure that the block is 4 periods of the modulation freq
     #  Ensure that the block is 4 periods of the modulation freq
     LutMan.M_block_length.set(960e-9)  # in ns
     LutMan.M_ampCW.set(0.4)
     LutMan.render_wave('M_ModBlock', time_unit='ns')
-    # divide instead of multiply by 1e-9 because of rounding errs
-    S1 = swf.UHFQC_Lutman_par_with_reload(LutMan,
-                                        LutMan.mixer_alpha,
-                                        ['M_ModBlock'], run=True, single=False)
-    S2 =swf.UHFQC_Lutman_par_with_reload(LutMan,
-                                         LutMan.mixer_phi,
-                                        ['M_ModBlock'], run=True, single=False)
+    # divide instead of multiply by 1e-9 because of rounding errors
+    S1 = swf.UHFQC_Lutman_par_with_reload(
+        LutMan, LutMan.mixer_alpha, ['M_ModBlock'], run=True, single=False)
+    S2 = swf.UHFQC_Lutman_par_with_reload(
+        LutMan, LutMan.mixer_phi, ['M_ModBlock'], run=True, single=False)
 
     detector = det.Signal_Hound_fixed_frequency(
-                SH, frequency=(source.frequency.get() -
-                               LutMan.M_modulation()),
-                Navg=5, delay=0.0, prepare_each_point=False)
+        SH, frequency=(source.frequency.get() -
+                       LutMan.M_modulation()),
+        Navg=5, delay=0.0, prepare_each_point=False)
 
     ad_func_pars = {'adaptive_function': nelder_mead,
-                        'x0': [1.0, 0.0],
-                        'initial_step': [.15, 10],
-                        'no_improv_break': 10,
-                        'minimize': True,
-                        'maxiter': 500}
+                    'x0': [1.0, 0.0],
+                    'initial_step': [.15, 10],
```

Comment:
I would change the no_improvement break to 15 as I have found 10 to be insufficient for reliable convergence.